addons/account/models/account_move.py

```diff
@@ -656,6 +660,16 @@ def _compute_date(self):
                 # might be protected because `_get_accounting_date` requires the `name`
                 self.env.add_to_compute(self._fields['name'], move)
 
+    @api.depends('move_type', 'journal_id')
+    def _compute_invoice_cash_rounding_id(self):
+        for record in self:
+            if record.invoice_cash_rounding_id:
+                record.invoice_cash_rounding_id = record.invoice_cash_rounding_id
```

Comment:
Avoid re-triggering dependencies
```suggestion
            if record.invoice_cash_rounding_id:
                continue
```